src/widgets/requirements/ui/requirements.module.scss

```diff
@@ -0,0 +1,54 @@
+.content {
+  display: flex;
+  flex-direction: column;
+  gap: 32px;
+
+  width: 100%;
+  max-width: $content-width;
+  padding: 40px 120px;
+
+  @include media-laptop {
+    padding: 40px 40px;
+  }
+
+  @include media-tablet {
+    padding: 40px 16px;
+  }
+}
```

Comment:
There is a global class content which is used in most widgets and sets the same paddings.  So, maybe it is better not to override it, but to rename the class differently, for example requirements, and remove all padding, as they will be set through the global class content. It goes something like this:

```
.requirements {
  display: flex;
  flex-direction: column;
  gap: 32px;

  width: 100%;
  max-width: $content-width;
}
```

```
<section className={cx('container')}>
      <article className={cx('content', 'requirements')}>
```

Reply:
Done